src/raft.c

```diff
@@ -210,7 +214,7 @@ static void executeLogEntry(RedisRaftCtx *rr, raft_entry_t *entry, raft_index_t
     rr->snapshot_info.last_applied_term = entry->term;
     rr->snapshot_info.last_applied_idx = entry_idx;
 
-    RaftRedisCommandArrayFree(&entry_cmds);
```

Comment:
do we want to free array here if its from the req?

Reply:
It was okay to call free even it's from the request, so we don't introduce another if check. 

Anyway, changed code a bit and did branching once, probably it's more clear right now.